### PR TITLE
Drop support for ``python setup.py test`` to run the tests.

### LINKED ISSUE #23: Drop support for `python setup.py test`
It is no longer used neither in `tox.ini` nor in `travis.yml`.

### PR DESCRIPTION
**Pull request by icemac**
Fixes #23.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,8 +40,6 @@ def alltests():
         include_package_data=True,
         install_requires=["ZConfig", "setuptools"],
         extras_require=dict(test=tests_require),
-        test_suite='__main__.alltests',
-        tests_require=tests_require
     )
 except ImportError:
     from distutils.core import setup
```
Comment (icemac) [inline setup.py:45]: Is this still needed? Do we need to support installing `zdaemon` without having setuptools installed?

Comment (mgedmin) [reply setup.py:45]: I'd say no.

**Review by mgedmin — APPROVED** [review]
LGTM

File: setup.py (hunk 1 above)
Comment (icemac) [reply setup.py]: I dropped it in https://github.com/zopefoundation/zdaemon/pull/26/commits/bf1875b7768dda75e37735768dc0d2ef60934ce4. Could you please take another look at my changes?

**Review by jamadden — APPROVED** [review]
LGTM.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**